docs/overview/deep_dive/ivy_tests.rst

```diff
@@ -354,7 +354,7 @@ The test then runs on the newly created arrays with specified data types.
 Why do we need helper functions
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-It is usually the case that any ivy function should run seamlessly on ‘all the possible varieties, as well as the edge cases’ encountered by the following parameters -:
+It is usually the case that any ivy function should run seamlessly on `all the possible varieties, as well as the edge cases` encountered by the following parameters -:
```

Comment:
Why do you think that is better?

Reply:
I thought that is what they were going for, and it was an accident that they did not italise it. What are your thoughts?

Reply:
I guess they were going for `'`, but they may have used a word processor that stylized the quotes. I like the stylized version more and see no need to remove it